pkg/arvo/app/launch.hoon

```diff
@@ -40,36 +41,55 @@
       path-to-tile=(map path @tas)
       first-time=?
   ==
++$  state-two
+  $:  %2
+      tiles=(set tile:launch)
+      data=tile-data:launch
+      path-to-tile=(map path @tas)
+      first-time=?
+      setup-who=_%.y
+  ==
 ::
 +$  card  card:agent:gall
 --
 ::
-=|  state-one
+=|  state-two
 =*  state  -
 %-  agent:dbug
 ^-  agent:gall
 |_  bol=bowl:gall
 +*  this  .
     def   ~(. (default-agent this %|) bol)
+    launch-who  [%pass /who %arvo %e %serve [~ /who] q.byk.bol /gen/who/hoon ~]
```

Comment:
Feels a bit weird to have this as an alias. I know the extra arm would make the agent core fail to nest, and launch doesn't use an extra core to work around that yet... Probably fine?

Reply:
could put in the type declaration core?

Reply:
That seems slightly better, yeah.

Reply:
That seems slightly better, yeah.